concepts/modularization/transition.md

```diff
@@ -0,0 +1,32 @@
+
+## Phase 2 - first module managed by `kyma-operator` integrated with KEB
+
+### Phase 2a - new clusters only (internal usage)
+
+`kyma-operator` is deployed in the control-plane and integrated with Kyma Environment Broker (KEB). Additional plan (preview) is created in KEB that creates Kyma clusters with kyma-operator only (no call to reconciler). This plan is the first playground to test KEB integration without affecting existing clusters and flows. The plan is tested with all environments (dev, stage, prod) to ensure we have all major integration issues resolved. The first module (frontrunner) can be submitted by the team and installed in the cluster. The minimal submission process is described and fully automated (manual steps are allowed only before submission). The minimal submission process looks like:
+- team verifies on their own if new module (or version) is product standard compliant 
+- team creates pull request to the control-plane that contains new/updated ModuleTemplate(s) (by doing it team declares product standard compliance and functional correctness - can be checkbox in PR template)
+- automated test verifies if the module operator can be installed/upgraded without issues for all release channels (with default settings) - it is not functional test or integration test 
+- automated test verifies if all images installed by operator are declared in the module template, signed by approved CA, vulnerability free
+- control-plane maintainer (codeowner) approves the PR which is automatically merged and rolled out
+
+
+### Phase 2b - move first module (frontrunner) from reconciler to kyma-operator (production)
+
+Regular plans use reconciler and kyma-operator in parallel (clusters are managed by both components, but their lists of modules don't overlap). Frontrunner module is disabled in the reconciler (ignored / replaced by dummy reconciler). `Kyma` resources containing the first module are populated for all existing clusters using script or KEB functionality. In each cluster `kyma-operator` installs frontrunner module operator with its CRD (configuration specification), and creates default configuration resource for the module. Module operator installs all needed components (including UI config map for Kyma Dashboard) and updated status of configuration custom resource (status.state = Ready). From that point in time cluster admin (or any user with sufficient roles) can edit the configuration resource to change module settings or can remove the module completely by deleting it from the list in the `Kyma` resource.
```

Comment:
Are "frontrunner" and "first" synonyms here? If yes, I suggest to keep just one term for better consistency.

```suggestion
Regular plans use reconciler and kyma-operator in parallel (clusters are managed by both components, but their lists of modules don't overlap). The frontrunner module is disabled in the reconciler (ignored or replaced by a dummy reconciler). `Kyma` resources containing the frontrunner module are populated for all existing clusters using a script or KEB functionality. In each cluster, `kyma-operator` installs the frontrunner module operator with its CRD (configuration specification), and creates a default configuration resource for the module. Module operator installs all needed components (including UI config map for Kyma Dashboard) and updates the status of the configuration custom resource (status.state = Ready). From that point in time, the cluster admin (or any user with sufficient authorization) can edit the configuration resource to change the module settings, or can remove the module completely by deleting it from the list in the `Kyma` resource.
```

I'm not sure I got the "updated" vs "updates" suggestion correctly; this sentence was a bit confusing.